apis/elbv2/v1alpha1/custom_types.go

```diff
@@ -0,0 +1,142 @@
+package v1alpha1
+
+import xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+
+// CustomTargetGroupTuple includes custom fields about target groups.
+// Only used with ForwardActionConfig to route to multiple target groups.
+type CustomTargetGroupTuple struct { // inject refs and selectors into TargetGroupTuple
+	// Provides information about how traffic will be
+	// distributed between multiple target groups in a forward rule.
+	TargetGroupTuple `json:",inline"`
+
+	// Reference to TargetGroupARN used to set TargetGroupARN
+	// +optional
+	TargetGroupARNRef *xpv1.Reference `json:"targetGroupARNRef,omitempty"`
```

Comment:
`json:"targetGroupArnRef,omitempty"`

Nitpick: while we don't have explicit convention around this, I think it's fair to go with calculated lower camel, i.e.` targetGroupArnRef`, instead of taking known words into account. There are some places we do one and other places we do the other, but I believe calculated lower camel is the most consistent for repos where there is code generation, like this one or Jet-based providers.

Reply:
Yeah, I mostly followed the AWS Go api naming scheme, but I personally prefer lower camel for the CRDs as I find it more readable.